### DISCUSSION TITLE
What's bloating my storage on self hosted setup?

### DISCUSSION THREAD

**Discussion by jorgnyg** [supabase/supabase#8684] [Questions] [answered]
Running supabase on my homelab setup with the provided [docker-compose](https://github.com/supabase/supabase/blob/master/docker/docker-compose.yml).
Something is constantly writing to the disk with the same interval, and I have a hard time figuring out what.

I know it's something related to supabase, because when I take down all the related docker services, the storage drops down to where it should be. So it seems like it's something non-persistent.

Before taking down related supabase docker services.
![diskuse](https://user-images.githubusercontent.com/35953480/187067288-e457d430-66ad-46d7-88c2-ea3bafb2a0a9.png)

After taking down related supabase docker services. 
![image](https://user-images.githubusercontent.com/35953480/187067391-3f4515fa-268d-4673-bb30-b672788f6949.png)

Anyone with an idea of what could be the root cause?

**Comment by GaryAustin1** [comment, marked as answer]
Maybe this https://github.com/supabase/supabase/issues/5023.   Log files.

**Reply by jorgnyg** [reply]
Disabled all logging just to test, and it stopped the bloating. Thanks!
